Tests/RxCocoaTests/UIPickerView+RxTests.swift

```diff
@@ -37,4 +38,271 @@ final class UIPickerViewTests: RxTest {
+    }
+    
+    func test_ModelSelected_itemTitles() {
+        var selectedItem: Int!
+        let pickerView = UIPickerView(frame: CGRect(x: 0, y: 0, width: 100, height: 50))
+        
+        _ = Observable.just([1, 2, 3]).bind(to: pickerView.rx.itemTitles) { _, _ in
+            return ""
+        }
+        _ = pickerView.rx.modelSelected(Int.self)
+            .subscribe(onNext: { item in
+                selectedItem = item
+            })
+        
+        pickerView.delegate!.pickerView!(pickerView, didSelectRow: 0, inComponent: 0)
+        
+        XCTAssertEqual(selectedItem, 1)
+    }
+    
+    func test_ModelSelected_itemAttributedTitles() {
+        var selectedItem: Int!
+        let pickerView = UIPickerView(frame: CGRect(x: 0, y: 0, width: 100, height: 50))
+        
+        _ = Observable.just([1, 2, 3]).bind(to: pickerView.rx.itemAttributedTitles) { _, _ in
+            return NSAttributedString()
+        }
+        _ = pickerView.rx.modelSelected(Int.self)
+            .subscribe(onNext: { item in
+                selectedItem = item
+            })
+        
+        pickerView.delegate!.pickerView!(pickerView, didSelectRow: 0, inComponent: 0)
+        
+        XCTAssertEqual(selectedItem, 1)
+    }
+    
+    func test_ModelSelected_items() {
+        var selectedItem: Int!
+        let pickerView = UIPickerView(frame: CGRect(x: 0, y: 0, width: 100, height: 50))
+        
+        _ = Observable.just([1, 2, 3])
+            .bind(to: pickerView.rx.items) { _, _, _ in
+                return UIView()
+            }
+        _ = pickerView.rx.modelSelected(Int.self)
+            .subscribe(onNext: { item in
+                selectedItem = item
+            })
+        
+        pickerView.delegate!.pickerView!(pickerView, didSelectRow: 0, inComponent: 0)
+        
+        XCTAssertEqual(selectedItem, 1)
+    }
+    
+    func test_ModelSelected_itemsWithCustomAdapter() {
+        var selectedItem: Int!
+        let pickerView = UIPickerView(frame: CGRect(x: 0, y: 0, width: 100, height: 50))
+        
+        _ = Observable.just([[1, 2, 3]])
+            .bind(to: pickerView.rx.items(adapter: TestPickerViewAdapter()))
+        _ = pickerView.rx.modelSelected(Int.self)
+            .subscribe(onNext: { item in
+                selectedItem = item
+            })
+        
+        pickerView.delegate!.pickerView!(pickerView, didSelectRow: 0, inComponent: 0)
+        
+        XCTAssertEqual(selectedItem, 1)
+    }
+    
+    func test_modelAtIdexPath_ThrowsError_itemTitles() {
+        let pickerView = UIPickerView(frame: CGRect(x: 0, y: 0, width: 100, height: 50))
+        
+        _ = Observable<[Int]>.never().bind(to: pickerView.rx.itemTitles) { _ in
+            return ""
+        }
+        
+        do {
+            let _: Int = try pickerView.rx.model(at: IndexPath(item: 0, section: 0))
+            XCTFail()
+        } catch let error as RxCocoaError {
+            XCTAssertTrue(error.isItemsNotYetBound)
```

Comment:
Not sure if it's not overkill to have assertion on specific error rather just that it throws